src/operations/paintLine.ts

```diff
@@ -0,0 +1,84 @@
+import { IJS } from '../IJS';
+import checkProcessable from '../utils/checkProcessable';
+import { getOutputImage } from '../utils/getOutputImage';
+
+export interface Point {
+  /**
+   * point row
+   *
+   */
+  row: number;
+  /**
+   * point column
+   *
+   */
+  column: number;
+}
+export interface PaintLineOptions {
+  /**
+   * Array of 3 elements (R, G, B),or 4 elements (R, G, B, A) default is red.
+   *
+   * @default [image.maxValue, 0, 0]
+   */
+  color?: [number, number, number] | [number, number, number, number];
+  /**
+   * Image to which the resulting image has to be put.
+   */
+  out?: IJS;
+}
+/**
+ * Paint a polyline defined by an array of points.
+ *
+ * @memberof Image
+ * @instance
+ * @param image - Image to process.
+ * @param from - Line starting point.
+ * @param to - Line ending point.
+ * @param options - Paint Line options.
+ * @returns The original painted image
+ */
+export function paintLine(
+  image: IJS,
+  from: Point,
+  to: Point,
+  options: PaintLineOptions = {},
+) {
+  let newImage = getOutputImage(image, options, { clone: true });
+  const { color = [newImage.maxValue, 0, 0] } = options;
```

Comment:
The default color must depend on the kind of image:
- Grey -> [0]
- GreyA -> [0, image.maxValue]
- RGB -> [0, 0, 0]
- etc.

You can create a function to generate it and do: `const { color = getDefaultColor(newImage) } = options;`

Reply:
fixed